spec/requests/reset_password_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe 'ResetPassword', type: :request do
+  let(:user) { create(:user) }
+
+  it 'get forgot password form' do
+    get '/forgot_password'
+    expect(response).to have_http_status(:success)
+  end
+
+  it 'get reset password form' do
+    get '/reset_password'
+    expect(response).to have_http_status(:success)
+  end
+
+  context 'right params when forgot password' do
```

Comment:
I think it is worth creating contexts for 'when forgot password', 'when reset password'